app/schemas/FOOD.json

```diff
@@ -0,0 +1,40 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "properties": {
+    "foodId": {
+      "type": "string"
+    },
+    "name": {
+      "type": "string"
+    },
+    "description": {
+      "type": "string"
+    },
+    "nutritionalData": {
+      "type": "object",
+      "properties": {
+        "calories": {
```

Comment:
how come calories are `integers` while protein, carbs, fats and fiber are `strings`? 

How do we want to store the unit? as a part of the string or in a separate field?

Reply:
the initial goal was to have it as a part of the string. eg "30g" for protein. However, we can have it as an integer both works